safedelete/models.py

```diff
@@ -160,6 +160,22 @@ def delete(self, force_policy=None, **kwargs):
             # soft-delete the object
             self.delete(force_policy=SOFT_DELETE, **kwargs)
 
+    @staticmethod
+    def has_unique_fields(model):
```

Comment:
Make it a `@classmethod` instead so you don't need to pass it the class twice.

Reply:
done.